src/main/java/com/amazon/ion/impl/bin/IonRawBinaryWriter_1_1.kt

```diff
@@ -327,13 +326,13 @@ class IonRawBinaryWriter_1_1 internal constructor(
 
     override fun writeFieldName(text: CharSequence) {
         confirm(currentContainer.type == Struct) { "Can only write a field name inside of a struct." }
-        if (!currentContainer.usesFlexSym) switchToFlexSym()
+        if (!currentContainer.usesFlexSym) switchCurrentStructToFlexSym()
 
         currentContainer.length += buffer.writeFlexSym(utf8StringEncoder.encode(text.toString()))
         hasFieldName = true
     }
 
-    private fun switchToFlexSym() {
+    private fun switchCurrentStructToFlexSym() {
```

Comment:
🗺️ Now that I've had a bit of time from writing this function, when I came back here and saw it, I thought the name was a bit too vague.